services/collectibles/api.go

```diff
@@ -131,6 +136,77 @@ func (api *API) DeployCollectibles(ctx context.Context, chainID uint64, deployme
 	return DeploymentDetails{address.Hex(), tx.Hash().Hex()}, nil
 }
 
+func (api *API) DeployOwnerToken(ctx context.Context, chainID uint64, ownerParameters DeploymentParameters, masterParameters DeploymentParameters, txArgs transactions.SendTxArgs, password string) (DeploymentDetails, error) {
```

Comment:
TO make it clearer. Also, calling MasterToken or TokenMaster just `Master` is confusing and if we ever have another type of Master, it would be super confusing
```suggestion
func (api *API) DeployOwnerToken(ctx context.Context, chainID uint64, ownerTokenParameters DeploymentParameters, masterTokenParameters DeploymentParameters, txArgs transactions.SendTxArgs, password string) (DeploymentDetails, error) {
```